libraries/Microsoft.Bot.Builder.Dialogs/InputRecognizer.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Schema;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.Dialogs
+{
+    public class InputRecognizer 
```

Comment:
>InputRecognizer [](start = 17, length = 15)

I'd prefer just plain old recognizer.  What else would a recognizer be doing?  input is a completely generic word that would cover any use of recognition.

Reply:
renamed

---
In reply to: [360619857](https://github.com/microsoft/botbuilder-dotnet/pull/3174#discussion_r360619857) [](ancestors = 360619857)